ts/packages/providers/langchain/src/index.ts

```diff
@@ -1,41 +1,47 @@
 /**
- * Langchain Provider
```

Comment:
Please don't let AI run wild, review the changes thoroughl. This shouldn't have changed.